code/game/machinery/vending.dm

```diff
@@ -1161,13 +1182,33 @@ var/global/num_vending_terminals = 1
 		/obj/item/weapon/glowstick/yellow = 2,
 		/obj/item/weapon/glowstick/magenta = 2
 		)
-	product_ads = "Only the finest!;Have some tools.;The most robust equipment.;The finest gear in space!"
+	product_ads = list(
+		"Only the finest!",
+		"Have some tools.",
+		"The most robust equipment.",
+		"The finest gear in space!"
+	)
 	pack = /obj/structure/vendomatpack/assist
 
 /obj/machinery/vending/coffee
 	name = "\improper Hot Drinks machine"
 	desc = "A vending machine that dispenses hot drinks."
-	product_ads = "Have a drink!;Drink up!;It's good for you!;Would you like a hot joe?;I'd kill for some coffee!;The best beans in the galaxy.;Only the finest brew for you.;Mmmm. Nothing like a coffee.;I like coffee, don't you?;Coffee helps you work!;Try some tea.;We hope you like the best!;Try our new chocolate!;Admin conspiracies"
+	product_ads = list(
+		"Have a drink!",
+		"Drink up!",
+		"It's good for you!",
+		"Would you like a hot joe?",
+		"I'd kill for some coffee!",
+		"The best beans in the galaxy.",
+		"Only the finest brew for you.",
+		"Mmmm. Nothing like a coffee.",
+		"I like coffee, don't you?",
+		"Coffee helps you work!",
+		"Try some tea.",
+		"We hope you like the best!",
+		"Try our new chocolate!",
+		"Admin conspiracies"
```

Comment:
This is horrifically out of place with the other ads for this machine.

Reply:
095e84de15d79898a76238f7e94c46df2b096851, although probably before that as well